Assets/World/Global.gd

```diff
@@ -1,5 +1,12 @@
 extends Node
 
+signal camera_rotate_left
+signal camera_rotate_right
+signal camera_zoom_in
+signal camera_zoom_out
+signal camera_max_zoom(flag)
+signal camera_min_zoom(flag)
```

Comment:
Let's write it this way for more clarity.
The type hint actually doesn't do anything right now but doesn't throw an error either, so it will be there for a future Godot version when this is counted in eventually.
```suggestion
signal camera_max_zoom(is_limit_reached: bool)
signal camera_min_zoom(is_limit_reached: bool)
```